http/swagger.yml

```diff
@@ -8589,7 +8589,9 @@ components:
         - note
```

Comment:
This list of required properties has gotten really long. It might help readability to change it to an array format, like:
`required: [type, queries, colors, shape, note, showNoteWhenEmpty, prefix, suffix, tickPrefix, tickSuffix, legend, decimalPlaces]`

Reply:
Got it! adjusted it!